pom.xml

```diff
@@ -12,7 +12,7 @@
 
   <groupId>org.folio</groupId>
   <artifactId>mod-quick-marc</artifactId>
-  <version>5.1.0-SNAPSHOT</version>
+  <version>5.1.1-SNAPSHOT</version>
```

Comment:
You don't need to change version of the module. 
You should bump specification interface version.

Reply:
ok, done